setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="qtmcop-docs",
```

Comment:
I can't remember, are we doing away with the "QTMCoP" branding and just calling it TAU docs now?
```suggestion
    name="tau-docs",
```